imagedephi/redact/tiff.py

```diff
@@ -27,9 +35,37 @@ class TiffMetadataRedactionPlan(RedactionPlan):
     file_format = FileFormat.TIFF
     image_path: Path
     tiff_info: TiffInfo
-    redaction_steps: dict[int, ConcreteMetadataRule]
+    metadata_redaction_steps: dict[int, ConcreteMetadataRule]
+    image_redaction_steps: dict[int, ConcreteImageRule]
     no_match_tags: list[tifftools.TiffTag]
 
+    @staticmethod
+    def is_tiled(ifd: IFD):
+        """Determine if an IFD represents a tiled image."""
+        tile_width_tag = tifftools.constants.get_or_create_tag("TileWidth", tifftools.constants.Tag)
```

Comment:
Why do you call `get_or_create_tag`?  Can't you just do `tile_width_tag = tifftools.Tag.TileWidth`?